web/job_runs_controller_test.go

```diff
@@ -20,11 +21,42 @@ type JobRun struct {
 	ID string `json:"id"`
 }
 
+func BenchmarkJobRunsController_Index(b *testing.B) {
+	app, cleanup := cltest.NewApplication()
+	defer cleanup()
+	j := setupJobRunsControllerIndex(b, app)
+
+	b.ResetTimer()
+	for n := 0; n < b.N; n++ {
+		resp := cltest.BasicAuthGet(app.Server.URL + "/v2/specs/" + j.ID + "/runs")
+		assert.Equal(b, 200, resp.StatusCode, "Response should be successful")
+	}
+}
+
 func TestJobRunsController_Index(t *testing.T) {
 	t.Parallel()
+
 	app, cleanup := cltest.NewApplication()
 	defer cleanup()
 
+	j := setupJobRunsControllerIndex(t, app)
+	resp := cltest.BasicAuthGet(app.Server.URL + "/v2/specs/" + j.ID + "/runs")
+
+	assert.Equal(t, 200, resp.StatusCode, "Response should be successful")
+	var respJSON JobRunsJSON
+	assert.Nil(t, json.Unmarshal(cltest.ParseResponseBody(resp), &respJSON))
+
+	jr, err := app.Store.JobRunsFor(j.ID)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Let's change this to `assert.Nil(t, err)` to be consistent with other tests and not rely on the "log" package.

Reply:
Just updated, thanks. This is the opposite of what I'm used to in Java land with it being bad practise to assert null, taking all these little points in.